### DISCUSSION TITLE
Afc control

### DISCUSSION THREAD

**Discussion by Mauri740** [dl9rdz/rdz_ttgo_sonde#397] [Q&A] [answered]
I don't understand why I can't get the http://rdzsonde.local/live.json command to control AFC for Frequency Offset. I have two ttgos, only one of them is able to connect with the command. I have an error, page is not found
Thank you

**Comment by hr-ru** [comment, marked as answer]
You have two running at the same time? And they use the same mDNS name? 

You can try using the IP address directly instead of "rdzsonde.local", or you can change the mDNS name in the settings of tone of the two. Using the IP address should always work.

**Comment by Mauri740** [comment]
with IP address everything works fine. Thank you
